internal/testing/e2e/consul.go

```diff
@@ -423,8 +423,8 @@ func ConsulClient(ctx context.Context) *api.Client {
 	return mustGetTestEnvironment(ctx).consulClient
 }
 
-func ConsulCA(ctx context.Context) []byte {
-	return mustGetTestEnvironment(ctx).ca
+func ConsulCA(ctx context.Context) string {
```

Comment:
Everything that used this func was casting to `string`, so I just did it at the source